Ficek_QuantiApp/Ficek_QuantiApp/Domain/Tests/RocketConvertersTests/RocketConverter+happyTests.swift

```diff
@@ -10,220 +10,220 @@ import ModelConvertible
-        }
-        
-        let sut =
-        Just(rocketDTO.diameter)
-            .setFailureType(to: RocketError.self)
-            .convertToDomainModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result = try! awaitPublisher(sut)
-        XCTAssertNoDifference(result, rocket.diameter)
-        
-        let sut2 =
-        Just(rocket.diameter)
-            .setFailureType(to: RocketError.self)
-            .convertToExternalModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result2 = try! awaitPublisher(sut2)
-        XCTAssertNoDifference(result2, rocketDTO.diameter)
+  let rocketDTO = RocketDTO.mock
+  let rocket = Rocket.mock
+  
+  override func invokeTest() {
+    withDependencies {
+      $0.heightConverter = .live()
+      $0.enginesConverter = .live()
+      $0.secondStageConverter = .live()
+      $0.firstStageConverter = .live()
+      $0.massConverter = .live()
+      $0.diameterConverter = .live()
+    } operation: {
+      super.invokeTest()
     }
+  }
+  func test_diameterConverter_success() {
     
-    func test_heightConverter_success() {
-        let converter = withDependencies { dependency in
-        } operation: {
-            HeightConverterKey.liveValue
-        }
-        
-        let sut =
-        Just(rocketDTO.height)
-            .setFailureType(to: RocketError.self)
-            .convertToDomainModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result = try! awaitPublisher(sut)
-        XCTAssertNoDifference(result, rocket.height)
-        
-        let sut2 =
-        Just(rocket.height)
-            .setFailureType(to: RocketError.self)
-            .convertToExternalModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result2 = try! awaitPublisher(sut2)
-        XCTAssertNoDifference(result2, rocketDTO.height)
+    let converter = withDependencies { dependency in
+    } operation: {
+      DiameterConverterKey.liveValue
     }
     
-    func test_massConverter_success() {
-        let converter = withDependencies { dependency in
-        } operation: {
-            MassConverterKey.liveValue
-        }
-        
-        let sut =
-        Just(rocketDTO.mass)
-            .setFailureType(to: RocketError.self)
-            .convertToDomainModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result = try! awaitPublisher(sut)
-        XCTAssertNoDifference(result, rocket.mass)
-        
-        let sut2 =
-        Just(rocket.mass)
-            .setFailureType(to: RocketError.self)
-            .convertToExternalModel(using: converter)
-            .eraseToAnyPublisher()
-        
-        let result2 = try! awaitPublisher(sut2)
-        XCTAssertNoDifference(result2, rocketDTO.mass)
+    let sut =
+    Just(rocketDTO.diameter)
+      .setFailureType(to: RocketError.self)
+      .convertToDomainModel(using: converter)
```

Comment:
Tyhle testy podle me delas uplne zbytecne, zatahujes do toho ten operator `convertToDomainModel`, ten uz je ale otestovanej. Proste si tu udelej instanci toho convertoru a testuj to normalne jako jednoduchou funkco, neni treba tu delat zadny asynchronni veci.

Reply:
Oprav pls tak vsechny testy na convertory.